.github/workflows/ci-all-testing.yml

```
@@ -24,9 +39,9 @@ jobs:
 
     - name: Install package
       run: |
-        pip install torch=="1.8.1+cpu" torchvision=="0.9.1+cpu" -f https://download.pytorch.org/whl/torch_stable.html
-        pip install mmcv-full=="1.3.2+torch.1.8.0+cpu" -f https://download.openmmlab.com/mmcv/dist/index.html --use-deprecated=legacy-resolver
-        pip install mmdet==2.12.0 --upgrade
+        pip install torch==${{ matrix.torch-version }} torchvision==${{ matrix.torchvision-version }} -f https://download.pytorch.org/whl/torch_stable.html
+        pip install mmcv-full==${{ matrix.mmcv-full-version }} -f https://download.openmmlab.com/mmcv/dist/cpu/torch${{ matrix.torch-version }}/index.html
+        pip install mmdet==${{ matrix.mmdet-version }}
         pip install -e ".[all,dev]"
         pip install yolov5-icevision --upgrade
```

> MMLabs have a pypi package (`openmim `) that automagically installs the right versions of both mmcv and mmdet without providing `torch` and `cuda` versions. That's how I'm using it:
> 
> ```
> pip install openmim -q
> mim install mmcv-full
> mim install mmdet
> ```

Thanks for chipping in @ai-fast-track! So do you suggest doing something like this? And do I need to add a `pip install fastai` line here (before `pip install -e .[all,dev]`)?

```suggestion
        pip install torch==${{ matrix.torch-version }} torchvision==${{ matrix.torchvision-version }} -f https://download.pytorch.org/whl/torch_stable.html
        pip install openmim
        mim install mmcv-full==${{ matrix.mmcv-full-version }}
        mim install mmdet==${{ matrix.mmdet-version }}
        pip install -e ".[all,dev]"
        pip install yolov5-icevision --upgrade
```